jvm-agent/src/main/org/jetbrains/lincheck/jvm/agent/analysis/controlflow/BasicBlockControlFlowGraph.kt

```diff
@@ -83,6 +83,23 @@ class BasicBlockControlFlowGraph(
     var loopInfo: MethodLoopsInformation? = null
         private set
 
+    /**
+     * Whether this CFG is reducible.
+     */
+    val isReducible: Boolean
```

Comment:
Let's just make it optional:
```
var isReducible: Boolean? = null
     private set
```

and return `null` in case when loop information has not yet been computed. 

And document that `isReducible` is known (!= null) only after `computeLoopInformation()` was called.